### DISCUSSION TITLE
Tensioner for events

### DISCUSSION THREAD

**Discussion by metalgearsloth** [space-wizards/space-station-14#4545] [Ideaguys]
Some people want it, some people don't want it (why I wrote the current one as it is). Ideally you'd be able to swap these out freely so servers could make their own with the same events but this is for bikeshedding how it works.

AFAIK most games like Rimworld or DF have a wealth value attached to things (e.g. number of alive players, size of the station, number of dead antags) and that determines the severity + types of events that can spawn.

Could also potentially use something like points buys where wealth accumulates over time (I think CEV-Eris does this?)

**Comment by PaulRitter** [comment]
it would probably be best to integrate events into the dynamic gamemode (i know for /vg/, unsure if other servers have it)

the tldr is: there is a threat-level similar to rimworld-esque wealth, that increases due to certain variables (players, size of station, dead antags, wealth, certain items, roundtime). all antags have a threat value that will get deducted from the total counter, so the gamemode basically "buys" antags continually with the threat-level.

i remember i had an experiement spreadsheet for this system laying around, feel free to reverse engineers or ping someone on the /vg/ codercord for details of the system: 
[Dynamic weight modificaiton simulation [SHARED].xlsx](https://github.com/space-wizards/space-station-14/files/7098830/Dynamic.weight.modificaiton.simulation.SHARED.xlsx)

**Comment by mishchenkoy** [comment]
🦁✅
